isbn/bl.php

```diff
@@ -50,22 +50,22 @@
 $error = yaz_error($id);
 if (!empty($error)) {
     echo "Error Number: " . yaz_errno($id);
-    echo "Error Description: " . $error ;
+    echo "Error Description: " . $error;
     echo "Additional Error Information: " . yaz_addinfo($id);
 }
 
 $outputString = "<?xml version=\"1.0\"?>\n";
 $outputString .= "<collection>\n";
 $outputArray = [];
 
-for ($p = 1; $p <= yaz_hits($id); $p++) {
+for ($p = 1; $p<=yaz_hits($id); $p++) {
```

Comment:
This change is surprising. In line 80 blanks are added around a comparison operator.

Reply:
Yes, indeed. I am open for suggestions...

Reply:
Maybe this is because of our [Scrutinizer settings](https://scrutinizer-ci.com/g/UB-Mannheim/malibu/settings/build-config), which might need to be adjusted:

![malibu-coding](https://user-images.githubusercontent.com/5199995/31574750-faa5e14a-b0d5-11e7-8f37-21beca8843d1.jpg)

I will undo these deletions of spaces around `<=` for this PR.

Reply:
Okay, I changed that setting now.